src/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+.navBar {
+    display: flex;
+    align-items: center;
+    background-color: #e9ecef;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    width: 100%;
+    border-radius: 0.7em;
+    margin-right: 1em;
+    height: 4em;
+  }
+  
+.navList {
+display: flex;
+align-items: center;
+list-style-type: none;
+}
+
+.navItem {
+margin: 0 1rem;
+cursor: pointer;
+transition: color 0.2s;
+font-family: Roboto;
+font-weight: 300;
+}
+
+.navItem:hover {
+color: #777;
+}
```

Comment:
Please make the indentations on everything.
I suggest you to find in the settings of VSCode the place where when you save the file, the ```prettier``` is working, so cases like this will not happen.